### DISCUSSION TITLE
How to control which tiny editor is used in editable block

### DISCUSSION THREAD

**Discussion by InfGame** [voidlabs/mosaico#693] [Q&A] [answered]
Just a question because i dont seem to really understand how to control it.

I have a template where there are some blocks i only want users to change if the text is bold/ italic or normal (the small editor type)

and then there is others where i want them to have more freedom with append links to words and such (the normal editor?)

and ther eis also a 3rd type of editor which i cant seem to get to where you are also able to paste variables into the textblock.

my question is how do i control which type of tiny editor is used.

**Comment by bago** [comment, marked as answer]
Mosaico automatically apply a simple editor to inline elements and a complete editor to block elements.

So, if your element is a TD or a DIV then you get a full editor, otherwise a "singleline" editor.

IIRC you can manually control this using the property data-ko-editor-style="multiline" or data-ko-editor-style="singleline".

**Comment by bago** [comment]
How the singleline and multiline are handled by tinymce is customized by `ko.bindingHandlers.wysiwyg.standardOptions` and `ko.bindingHandlers.wysiwyg.fullOptions`.
